# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/pkg_patch/pkg_perl.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2004 Clifford Wolf
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Yes I know that patch is suboptimal. It should do for now ;)

--- perl-5.8.0/ext/Errno/Errno_pm.PL.orig	2003-09-20 18:41:01.000000000 +0200
+++ perl-5.8.0/ext/Errno/Errno_pm.PL	2003-09-20 18:41:31.000000000 +0200
@@ -109,7 +109,7 @@
 	     ) {
 	# Some Linuxes have weird errno.hs which generate
 	# no #file or #line directives
-	$file{'/usr/include/errno.h'} = 1;
+	$file{'/usr/dietlibc/include/errno.h'} = 1;
     } elsif ($^O eq 'MacOS') {
 	# note that we are only getting the GUSI errno's here ...
 	# we might miss out on compiler-specific ones
